src/test/java/api/requests/scenarios/MoveRequestPolicyTests.java

```diff
@@ -0,0 +1,419 @@
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.ISODateTimeFormat;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.LoanPolicyBuilder;
+import api.support.builders.MoveRequestBuilder;
+import api.support.builders.NoticeConfigurationBuilder;
+import api.support.builders.NoticePolicyBuilder;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Notes:<br>
+ *  MGD = Minimum guaranteed due date<br>
+ *  RD = Recall due date<br>
+ */
+public class MoveRequestPolicyTests extends APITests {
+  private static final String RECALL_TO_LOANEE = "Recall loanee";
+  private static Clock clock;
+
+  private NoticePolicyBuilder noticePolicy;
+
+  @BeforeClass
+  public static void setUpBeforeClass() {
+    clock = Clock.fixed(Instant.now(), ZoneOffset.UTC);
+  }
+
+  @Before
+  public void setUp() {
+    // reset the clock before each test (just in case)
+    ClockManager.getClockManager().setClock(clock);
+  }
+
+  @Before
+  public void setUpNoticePolicy() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    UUID recallToLoaneeTemplateId = UUID.randomUUID();
+    JsonObject recallToLoaneeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(recallToLoaneeTemplateId)
+      .withEventType(RECALL_TO_LOANEE)
+      .create();
+
+    noticePolicy = new NoticePolicyBuilder()
+      .withName("Policy with recall notice")
+      .withLoanNotices(singletonList(recallToLoaneeConfiguration));
+
+    useLoanPolicyAsFallback(
+      loanPoliciesFixture.canCirculateRolling().getId(),
+      requestPoliciesFixture.allowAllRequestPolicy().getId(),
+      noticePoliciesFixture.create(noticePolicy).getId());
+  }
+
+  @Test
+  public void cannotMoveRecallRequestsWithRequestPolicyNotAllowingHolds()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final String anyNoticePolicy = noticePoliciesFixture.activeNotice().getId().toString();
+    final String anyLoanPolicy = loanPoliciesFixture.canCirculateRolling().getId().toString();
+    final String bookMaterialType = materialTypesFixture.book().getId().toString();
+    final String anyRequestPolicy = requestPoliciesFixture.allowAllRequestPolicy().getId().toString();
+
+    ArrayList<RequestType> allowedRequestTypes = new ArrayList<>();
+    allowedRequestTypes.add(RequestType.RECALL);
+    allowedRequestTypes.add(RequestType.PAGE);
+    final String noHoldRequestPolicy = requestPoliciesFixture.customRequestPolicy(allowedRequestTypes,
+      "All But Hold", "All but Hold request policy").getId().toString();
+
+    //This rule is set up to show that the fallback policy won't be used but the material type rule m is used instead.
+    //The material type rule m allows any patron to place any request but HOLDs on any BOOK, loan or notice types
+    final String rules = String.join("\n",
```

Comment:
Why does it matter that it isn't the fallback policy that is applied?

Reply:
I am not familiar with policies and copied from another test.

Reply:
Could you please assist in either understanding the policy rules or let me know what I need to change?

Reply:
I'm not really sure how to advise.

What is the test trying to demonstrate? Does that rely on complex circulation rules?

Most of the time, I use the `useLoanPolicyAsFallback` method in `APITests` as that is the simplest way of specifying the policies to use, without needing to understand the rules themselves.

As to the rules, I am far from an expert, the best place to start is [the documentation](https://github.com/folio-org/mod-circulation/blob/master/doc/circulationrules.md)

Apologies that this response is rather rushed

Reply:
The test is another scenario, [UIREQ-269](https://issues.folio.org/browse/UIREQ-269) scenario 4, which is to prevent moving recall request when policy disallowing holds.

Reply:
Sure, do you know why this policy cannot be the fallback policy?

Reply:
I do not. As I said, the majority of this was from another test to produce the expected failure before writing the feature.

Reply:
Ok, we likely need to come back to this in the future